src/gasman.c

```diff
@@ -1883,88 +1875,6 @@ UInt CollectBags (
 
     /* * * * * * * * * * * * * * * sweep phase * * * * * * * * * * * * * * */
 
-#if 0
-    /* call freeing function for all dead bags                             */
-    if ( NrTabFreeFuncBags ) {
```

Comment:
I suspected that it *could* have been useful, but quite obviously was unused up to now. 
If we were to introduce a finalizer API, we'd probably reinvent this stuff, so now's maybe the time to make that decision?